src/jupyterhub_config/custom_docker_spawner.py

```diff
@@ -49,6 +51,29 @@ def _ensure_user_directory(self, user_dir: Path, username: str):
         else:
             self.log.info(f'Reusing user directory for {username}')
 
+    def _ensure_bashrc(self, user_dir: Path):
+        """
+        Ensure the user's .bashrc and .bash_profile files exist, copying them from .tmpl templates if needed.
+        """
+
+        config_dir = Path(os.environ['CONFIG_DIR'])
+        bashrc_ext_src = config_dir / '.bashrc.tmpl'
+        bash_profile_ext_src = config_dir / '.bash_profile.tmpl'
+
+        shutil.copy2(bashrc_ext_src, user_dir/'.bashrc.tmpl')
+        shutil.copy2(bash_profile_ext_src, user_dir/'.bash_profile.tmpl')
+
+        bashrc_dest = user_dir / '.bashrc'
+        bash_profile_dest = user_dir / '.bash_profile'
+
+        if not bashrc_dest.exists():
+            self.log.info(f'Creating .bashrc file for {user_dir}')
+            shutil.copy2(bashrc_ext_src, bashrc_dest)
```

Comment:
Why copy twice? Seems like just copying from the config dir directly to *_dest would work

Reply:
Can you add a comment to that effect? Will probably confuse future ppl working on this and they might simplify things w/o realizing your intent

Reply:
👍